### PR TITLE
Merging errors were fixed and extra improvements were made in 'btp-cf-buildpacks-python-create'

### PR DESCRIPTION
**Pull request by Joysie**

### CODE REVIEW

File: tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md [hunk 1]
```diff
@@ -117,11 +122,7 @@ Now you'll create a simple Python application.
 
     > Make sure you always execute `cf push` in the directory where the `manifest.yml` is located.
 
-7.	When the staging and deployment steps are complete, you can check the state and the URL of your application. Execute:
-
-    ```Bash
-    cf app mynewapp
-    ```
+7.  When the staging and deployment steps are complete, the `mynewapp` application should be successfully started and its details - displayed in the command console.
```
Comment (MichaelCzcz) [inline tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md:125]: Why the dash?

File: tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md [hunk 2]
```diff
@@ -378,11 +380,8 @@ Authentication in the SAP BTP, Cloud Foundry environment is provided by the Auth
 
     >As of this point of the tutorial, the URL of the `myweb` application will be requested instead of the `mynewapp` URL. It will then forward the requests to the `mynewapp` application.
 
-13.	Find the URL of the `web` application. Execute:
 
-    ```Bash
-    cf app web
-    ```
+13.	When the staging and deployment steps are complete, the `myweb` application should be successfully started and its details - displayed in the command console.
```
Comment (MichaelCzcz) [inline tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md:384]: Why the dash?

File: tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
 author_name: Gergana Tsakova
 author_profile: https://github.com/Joysie
-title: Create a Python Application via Cloud Foundry CLI
-description: Create a simple Python application in CF CLI and enable services for it.
+title: Create a Python Application via CF CLI
```
Comment (MichaelCzcz) [inline tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md:4]: Use the full name "Cloud Foundry Command Line Interface".

File: tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
+description: Create a simple Python application in the Cloud Foundry Command Line Interface (cf CLI) and enable services for it.
```
Comment (MichaelCzcz) [inline tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md:5]: After this step you can use the abbreviation "CF CLI". Please update the tutorial accordingly.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by Joysie** [comment]
Comments are applied. Thank you!

File: tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md [hunk 5]
```diff
@@ -27,13 +30,13 @@ primary_tag: software-product>sap-btp--cloud-foundry-environment
   - How to run authorization checks for your app
 
 
-This tutorial will guide you through creating and setting up a simple Python application by using the Cloud Foundry command line interface ( **cf CLI**). You will start by building and deploying a Web application that returns sample data – a **Hello World!** message, and then invoking this app through another one - a Web microservice (application router).
+This tutorial will guide you through creating and setting up a simple Python application by using cf CLI. You will start by building and deploying a Web application that returns sample data – a **Hello World!** message, and then invoking this app through another one - a Web microservice (application router).
```
Comment (MichaelCzcz) [inline tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md:33]: CF not cf. It's a product name.

File: tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md [hunk 6]
```diff
@@ -12,11 +12,14 @@ primary_tag: software-product>sap-btp--cloud-foundry-environment
 ## Prerequisites
  - You have registered for a Cloud Foundry [trial account] (hcp-create-trial-account) on SAP Business Technology Platform.
  - [Python] (https://www.python.org/downloads/) version 3.6.x or higher is installed locally. In this tutorial, we use Python **3.9.6**.
- - The [Cloud Foundry command line interface] (https://help.sap.com/products/BTP/65de2977205c403bbc107264b8eccf4b/4ef907afb1254e8286882a2bdef0edf4.html) is installed locally.
- - You have [npm] (https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) installed locally.
+ - [cf CLI] (https://help.sap.com/products/BTP/65de2977205c403bbc107264b8eccf4b/4ef907afb1254e8286882a2bdef0edf4.html) is installed locally.
```
Comment (MichaelCzcz) [inline tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md:15]: CF not cf. It's a product name.

File: tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md [hunk 7]
```diff
@@ -1,8 +1,8 @@
 ---
 author_name: Gergana Tsakova
 author_profile: https://github.com/Joysie
-title: Create a Python Application via Cloud Foundry CLI
-description: Create a simple Python application in CF CLI and enable services for it.
+title: Create a Python Application via Cloud Foundry Command Line Interface
+description: Create a simple Python application in the Cloud Foundry Command Line Interface (cf CLI) and enable services for it.
```
Comment (MichaelCzcz) [inline tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md:5]: CF not cf. It's a product name.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md (hunk 6 above)
Comment (Joysie) [reply tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md:15]: Hi Michael,

I like CF better too. But in the official Cloud Foundry documentation, they always write it **cf CLI**:
https://docs.cloudfoundry.org/cf-cli/install-go-cli.html

Same with our (SAP) official documentation:
https://help.sap.com/products/BTP/65de2977205c403bbc107264b8eccf4b/4ef907afb1254e8286882a2bdef0edf4.html

File: tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md (hunk 7 above)
Comment (Joysie) [reply tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md:5]: See comment above.

File: tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md (hunk 5 above)
Comment (Joysie) [reply tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md:33]: See comments below.

**Pull request merged**